pkg/consensus/consensus.go

```diff
@@ -394,13 +393,13 @@ func (s *service) getBlock(h util.Uint256) block.Block {
 func (s *service) getVerifiedTx(count int) []block.Transaction {
 	pool := s.Config.Chain.GetMemPool()
 
-	var txx []mempool.TxWithFee
+	var txx []*transaction.Transaction
```

Comment:
Maybe we can declare a slice of `block.Transaction` here? And make no copy at the end.

Reply:
That was my thought too, but there are cases (and arguably more popular than the one related to `lastProposal`) when we take transactions from the mempool, so we have to have some common type. It can probably be optimized anyway, but it's not a priority for now, I think.